tests/end2end/playwright/editable-features.spec.js

```diff
@@ -0,0 +1,65 @@
+import { test, expect } from '@playwright/test';
+import {ProjectPage} from "./pages/project";
+import { expectParametersToContain, getAuthStorageStatePath } from './globals';
+
+test.describe('Editable features',
+    {
+        tag: ['@readonly'],
+    },()=> {
+        test('Check attribute table editable features - anonymous', async ({ page }) => {
+            const project = new ProjectPage(page, 'attribute_table_editable_features');
+            await project.open();
+
+            await project.openAttributeTable('filter_layer_by_user');
+
+            const tableHtml = project.attributeTableHtml('filter_layer_by_user');
+
+            await expect(tableHtml.locator('tbody tr')).toHaveCount(3);
+
+            for (let i = 1; i<=3; i++) {
+                let tr = tableHtml.locator('tbody tr[id="'+i+'"]');
+                await expect(tr.locator('lizmap-feature-toolbar .feature-edit')).toBeHidden()
+            }
+        })
+
+        test('Check attribute table editable features - user_in_group_a', async ({ browser }) => {
+            // login with specific user
+            const userContext = await browser.newContext({storageState: getAuthStorageStatePath('user_in_group_a')});
+            const userPage = await userContext.newPage();
+
+            const project = new ProjectPage(userPage, 'attribute_table_editable_features');
+            await project.open();
+
+
+            const getEditableFeaturesRequestPromise = project.waitForEditableFeaturesRequest();
+            await project.openAttributeTable('filter_layer_by_user');
+
+            const getEditableFeaturesRequest = await getEditableFeaturesRequestPromise;
+            const expectedParameters = {
+                'layerId': 'filter_layer_by_user_c733b070_38ab_425b_8c52_ae331633dcc6',
+                'features': '"gid" = \'1\' OR "gid" = \'3\' OR "gid" = \'2\'',
+            };
+            await expectParametersToContain('Editable features on attribute table - anonymous', getEditableFeaturesRequest.postData() ?? '', expectedParameters);
+            const response = await getEditableFeaturesRequest.response();
+
+            // check response
+            expect(response?.ok()).toBeTruthy();
+            expect(response?.status()).toBe(200);
+            // check content-type header, content is streamed by the server
+            expect(response?.headers()['content-type']).toContain('application/octet-stream');
```

Comment:
The response for editable features request has to be `application/json`. The controller has to be enhanced with `$rep->mimeType = 'application/json';`

Once done the expect response fixture could be use.